tests/functional/object/mcg/test_noobaa_metrics.py

```
@@ -18,6 +19,31 @@
+def wait_for_num_objects(mcg_obj, bucket_name, expected_count, timeout=360):
+    """
+    Poll bucket status until the number of objects reaches the expected count.
+
+    Args:
+        mcg_obj (obj): An object representing the current state of the MCG in the cluster
+        bucket_name (str): Name of the bucket to check
+        expected_count (int): Expected number of objects
+        timeout (int): Maximum time to wait in seconds
+
+    """
+    logger.info(f"Waiting for object count to reach {expected_count}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)